packages/web-components/fast-components/src/search/search.styles.ts

```diff
@@ -82,11 +103,36 @@ export const searchStyles: (
     }
 
     .clear-button {
-        position: absolute;
-        right: 0;
-        top: 1px;
+        margin: 1px;
         height: calc(100% - 2px);
         opacity: 0;
+        background: transparent;
+        color: ${neutralForegroundRest};
+        fill: currentcolor;
+        border: none;
+        border-radius: calc(${controlCornerRadius} * 1px);
+        min-width: calc(${heightNumber} * 1px);
+        font-size: ${typeRampBaseFontSize};
+        line-height: ${typeRampBaseLineHeight};
+        outline: none;
+        font-family: ${bodyFont};
+        padding: 0 calc((10 + (${designUnit} * 2 * ${density})) * 1px);
+    }
+
+    .clear-button:hover {
```

Comment:
Nit: We're calling this `clear-button` for the class, but `close-button` for the recipe. Can we update the recipe to match this?

Reply:
absolutely